roaringbitmap/src/main/java/org/roaringbitmap/longlong/Roaring64NavigableMap.java

```diff
@@ -159,6 +162,10 @@ public Roaring64NavigableMap(boolean signedLongs, boolean cacheCardinalities,
     resetPerfHelpers();
   }
 
+  public void setSerializationMode(int mode) {
```

Comment:
It is probably better to have a setter. Though:

- it sets a static field: the method should be static too
- it definitely needs additional javadoc, in the vein of the field documentation.

Reply:
may discuss in next CR thread